packages/composables/src/hooks/useCart/index.ts

```diff
@@ -153,6 +154,11 @@ export const useCart = (rootContext: ApplicationVueContext): IUseCart => {
     return cartPrice || 0;
   });
 
+  const shippingTotal = computed(() => {
+    const shippingTotal = cart?.value?.deliveries[0]?.shippingCosts?.totalPrice;
```

Comment:
Is optional chaining allowed?

Reply:
definitely you can, in composables the optional chaining is supported. in this case you can even use ....deliveries?.[0]? for arrays :+1: 

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining#array_item_access_with_optional_chaining

Reply:
Alright, cool!